_content/papers/buddha-and-metaphysics_wijesekera.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "The Buddha and Metaphysics"
+authors:
+  - wijesekera
+editor: "M. H. F. Jayasuriya"
+external_url: "https://archive.org/details/in.ernet.dli.2015.463583/page/n3/mode/2up"
+drive_links:
+  - "https://drive.google.com/file/d/19rYXVNrPkyrvmdrSTovUrXR6-17Sbdp4/view?usp=drive_link"
+formats: [pdf]
+course: view
```

Comment:
Hmmm  Rereading this, I'm wondering if it belongs here or shouldn't "fall down" to `cosmology` 🤔  What do you think?